go.mod

```diff
@@ -30,4 +30,5 @@ require (
 replace (
 	k8s.io/cli-runtime v0.20.4 => k8s.io/cli-runtime v0.21.0
 	k8s.io/kubectl v0.20.4 => k8s.io/kubectl v0.21.0
+	sigs.k8s.io/kustomize/kyaml => github.com/mengqiy/kustomize/kyaml v0.0.0-20210604212954-b6df45d28a4e
```

Comment:
I will update this once https://github.com/kubernetes-sigs/kustomize/pull/3965 is merged.

Reply:
We can do this in a separate PR.